### DISCUSSION TITLE
🚀 File Upload from Entry - Choosing Existing default to previously used Folder

### DISCUSSION THREAD

**Discussion by ic360-com** [ExpressionEngine/ExpressionEngine#4292] [Feature Requests]
### Suggest an idea for ExpressionEngine

Would it be possible to automatically navigate to the previously accessed Folder, when using the "Choose Existing" option to upload files/images when editing an Entry?
It's painful and laborious to navigate the nested Folder structure every time to get to the same place.

EE 7.4.9

**Comment by intoeetive** [comment]
@Yulyaswan perhaps there's an easy way for us to achieve that? Like using the browser's localStorage to remember last selection?

**Comment by Yulyaswan** [comment]
PR - https://github.com/ExpressionEngine/ExpressionEngine/pull/4488
